core/lib/entity-classification.js

```diff
@@ -0,0 +1,52 @@
+import {Audit} from '../audits/audit.js';
+import UrlUtils from './url-utils.js';
+import {EntityClassification as ComputedEntityClassification} from '../computed/entity-classification.js';
```

Comment:
our current pattern is to only rename the imports of the computed things if there is a naming conflict. There is no such conflict for this file